### PR TITLE
add the option to delete an uploaded image and change fileUploader design in the survey pageForm

### PR DESCRIPTION
**Pull request by MilanBarande**
[Link to the JIRA ticket](https://bluenove.atlassian.net/browse/DEVAS-1373)

### CODE REVIEW

**Comment by cedricmessiant** [comment]
just added the back-end part for the image deletion. @swergas @vincentfretin a little review of this part would be nice

File: assembl/static2/js/app/components/common/fileUploader.jsx [hunk 1]
```diff
@@ -116,9 +124,28 @@ class FileUploader extends React.Component<Object, FileUploaderProps, FileUpload
...
+            </OverlayTrigger>
           </div>
-        ) : null}
-        {!isImage && <div className="preview-title">{this.state.fileName}</div>}
+        ) : (
+          <div className={fileSrc && isImage ? 'preview' : 'hidden'}>
+            <img
+              src={fileSrc}
+              ref={(p) => {
+                this.preview = p;
```
Comment (swergas) [inline assembl/static2/js/app/components/common/fileUploader.jsx]: this.preview seems to be not used in the rest of the component

File: assembl/static2/tests/unit/components/common/__snapshots__/fileUploader.spec.jsx.snap [hunk 2]
```diff
@@ -47,6 +47,14 @@ exports[`FileUploader component should render a file uploader with the filename
       Choose a file to upload
     </span>
   </button>
+  <div
+    className="hidden"
+  >
+    <img
+      alt="preview"
+      src={undefined}
```
Comment (swergas) [inline assembl/static2/tests/unit/components/common/__snapshots__/fileUploader.spec.jsx.snap]: is it undefined on purpose?

File: assembl/static2/js/app/components/common/fileUploader.jsx (hunk 1 above)
Comment (MilanBarande) [reply assembl/static2/js/app/components/common/fileUploader.jsx]: @cedricmessiant is there a purpose for this ref ?

File: assembl/static2/tests/unit/components/common/__snapshots__/fileUploader.spec.jsx.snap (hunk 2 above)
Comment (MilanBarande) [reply assembl/static2/tests/unit/components/common/__snapshots__/fileUploader.spec.jsx.snap]: Yes, this snapshot is for this test: ```FileUploader component should render a file uploader with the filename if the file is not an image (fileOrUrl is a File object)```

**Comment by vincentfretin** [comment]
backend is ok

File: assembl/static2/js/app/components/common/fileUploader.jsx (hunk 1 above)
Comment (cedricmessiant) [reply assembl/static2/js/app/components/common/fileUploader.jsx]: I guess that there was a long time ago but it does not seem to be the case anymore. Can't find when that changed with blame and history so I guess we can remove it if everything seems fine

Comment (MilanBarande) [reply assembl/static2/js/app/components/common/fileUploader.jsx]: Ok I'll remove it

**Pull request merged**